model/artifact/artifact_file.go

```diff
@@ -1,10 +1,19 @@
 package artifact
 
 import (
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 )
 
 const Collection = "artifact_files"
+const PresignExpireTime = 15 * time.Minute
```

Comment:
this seems like it should be configurable, (and also somewhat longer.) I'm not sure if project or global makes the most sense.

Reply:
Configurable by the user?  It's not a big deal if it expires. All they have to do is refresh the page and get a new link.

Reply:
I updated it to expire after 24 hours